test/autoCompletion5.test.ts

```diff
@@ -0,0 +1,116 @@
+ *--------------------------------------------------------------------------------------------*/
+import { TextDocument } from 'vscode-languageserver';
+import { getLanguageService } from '../src/languageservice/yamlLanguageService';
+import { schemaRequestService, workspaceContext }  from './utils/testHelper';
+import { getLineOffsets } from '../src/languageservice/utils/arrUtils';
+import assert = require('assert');
+import path = require('path');
+import {toFsPath} from "./utils/testHelper";
+const languageService = getLanguageService(schemaRequestService, workspaceContext, [], null);
+
+const languageSettings = {
+    schemas: [],
+    completion: true
+};
+
+const uri = toFsPath(path.join(__dirname, './fixtures/testArrayIndent.json'));
+const fileMatch = ['*.yml', '*.yaml'];
+languageSettings.schemas.push({ uri, fileMatch: fileMatch });
+languageService.configure(languageSettings);
+
+suite('Auto Completion Tests', () => {
+
+    describe('yamlCompletion with array object', function () {
+
+        describe('doComplete', function () {
+
+            function setup(content: string) {
+                return TextDocument.create('file://~/Desktop/vscode-k8s/test.yaml', 'yaml', 0, content);
+            }
+
+            function parseSetup(content: string, position) {
+                const testTextDocument = setup(content);
+                return completionHelper(testTextDocument, testTextDocument.positionAt(position));
+            }
+
+            it('Indent should be considered with position relative to slash', done => {
+                const content = 'install:\n  - he';
+                const completion = parseSetup(content, content.lastIndexOf('he') + 2);
+                completion.then(function (result) {
+                    assert.equal('helm:\n  \tname: $1', result.items[0].insertText);
+                }).then(done, done);
+            });
+
+            it('Large indent should be considered with position relative to slash', done => {
+                const content = 'install:\n -            he';
+                const completion = parseSetup(content, content.lastIndexOf('he') + 2);
+                completion.then(function (result) {
+                    assert.equal('helm:\n             \tname: $1', result.items[0].insertText);
+                }).then(done, done);
+            });
+
+             it('Tab indent should be considered with position relative to slash', done => {
+                const content = 'install:\n -\t             he';
+                const completion = parseSetup(content, content.lastIndexOf('he') + 2);
+                completion.then(function (result) {
+                    assert.equal('helm:\n \t             \tname: $1', result.items[0].insertText);
+                }).then(done, done);
+            });
+
+        });
+    });
+});
+
+function is_EOL(c) {
+    return (c === 0x0A/* LF */) || (c === 0x0D/* CR */);
+}
+
+function completionHelper(document: TextDocument, textDocumentPosition) {

```

Comment:
If I'm not mistaken you shouldn't need this anymore, it now comes standard inside of the [YAMLCompletion]( https://github.com/redhat-developer/yaml-language-server/blob/master/src/languageservice/services/yamlCompletion.ts#L65)